blazy/blazy.d.ts

```diff
@@ -0,0 +1,72 @@
+// Type definitions for bLazy v1.5.2
+// Project: https://github.com/dinbror/blazy
+// Definitions by: Julien Paroche <https://github.com/julienpa>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/blazy
+// Definitions based on: http://dinbror.dk/blog/blazy
+
+declare var Blazy: Blazy;
+
+interface Blazy {
+
+  new (options: IBlazyOptions): IBlazyInstance;
+
+}
+
+interface IBlazyOptions {
```

Comment:
please remove `I` prefix from interface.
from http://www.typescriptlang.org/Handbook#writing-dts-files 

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.